gems/bundled_gems

```diff
@@ -11,6 +11,6 @@ net-pop         0.1.2   https://github.com/ruby/net-pop
 net-smtp        0.3.3   https://github.com/ruby/net-smtp
 matrix          0.4.2   https://github.com/ruby/matrix
 prime           0.1.2   https://github.com/ruby/prime
-rbs             2.8.3   https://github.com/ruby/rbs
+rbs             3.0.0.dev.1   https://github.com/ruby/rbs
```

Comment:
```suggestion
rbs             2.8.3  https://github.com/ruby/rbs 084fdb86387396eee08b2e525d648cae04d7b5e6
```

Reply:
I picked https://github.com/ruby/rbs/pull/1188/commits/84526b58a88818661541f10de40074d63f298541 it should be enough